packages/expect/src/matchers.ts

```diff
@@ -252,7 +251,7 @@ const matchers: MatchersObject = {
     return {message, pass};
   },
 
-  toBeLessThan(received: number, expected: number) {
+  toBeLessThan(this: any, received: number, expected: number) {
```

Comment:
Do you remember the reason for `this: any` argument?

`toBeGreaterThan` nor `toBeGreaterThanOrEqual` don’t have it even though their bodies are very similar.

If it is needed, it might be `MatcherState` type exported from `jest-types` package.

Reply:
It should use `MatcherState` (which might need to be tweaked)

Reply:
I've updated the matchers to use this as `MatcherState` and to type options as `MatcherHintOptions`.